metron-platform/metron-common/src/main/java/org/apache/metron/common/dsl/functions/DataStructureFunctions.java

```diff
@@ -137,6 +138,105 @@ public Object apply(List<Object> args) {
     }
   }
 
+  @Stellar( namespace="HLLP"
+          , name="CARDINALITY"
+          , description="Returns HyperLogLogPlus-estimated cardinality for this set"
+          , params = { "hyperLogLogPlus - the hllp set" }
+          , returns = "Long value representing the cardinality for this set"
+  )
+  public static class HLLPCardinality extends BaseStellarFunction {
+
+    @Override
+    public Object apply(List<Object> args) {
+      if (args.size() < 1) {
+        throw new IllegalArgumentException("Must pass an hllp set to get the cardinality for");
+      }
+      return ((HyperLogLogPlus) args.get(0)).cardinality();
+    }
+  }
+
+  @Stellar( namespace="HLLP"
+          , name="INIT"
+          , description="Initializes the set"
+          , params = {
+                      "p (required) - the precision value for the normal set"
```

Comment:
Are their any documents that we can link to describe the tradeoffs for these values?  I'm thinking of something like [here](https://en.wikipedia.org/wiki/File:Bloom_filter_fp_probability.svg) discussing the tradeoff between accuracy and size.